components/brave_shields/browser/domain_block_navigation_throttle.cc

```diff
@@ -122,6 +126,12 @@ DomainBlockNavigationThrottle::WillStartRequest() {
   if (tab_storage->IsProceeding())
     return content::NavigationThrottle::PROCEED;
 
+  // If this URL is about to be debounced, don't show an interstitial.
+  GURL debounced_url;
+  if (debounce_service_ &&
```

Comment:
can we accomplish the same thing by just reordering them so debounce is before domain block?

Reply:
@pilgrim-brave were you able to try this approach? any reason it wouldn't work?